app/.storybook/main.js

```diff
@@ -44,18 +44,6 @@ const config = {
       use: [
         "style-loader",
         { loader: "css-loader", options: { url: false } }, // this mirrors next.js behavior
-        {
-          loader: "string-replace-loader",
-          options: {
-            // Support deploying Storybook to a subdirectory (like GitHub Pages).
-            // This adds the BASE_PATH to the beginning of all relative URLs in the CSS.
-            // It handles relative urls, whether they are quoted or not.
-            search: /url\(("?)\//g,
-            replace(match, p1, offset, string) {
-              return `url(${p1}${BASE_PATH}/`;
-            },
```

Comment:
oh this is kind of clever. for my own understanding, why did you choose to get rid of this find/replace approach with the more explicit expose a function that the sass files need to call approach?

Reply:
This relied on a Webpack dependency, which would have required adding a custom Webpack configuration to Next.js. My impression though is Next.js has been [moving away from Webpack](https://nextjs.org/docs/advanced-features/turbopack) as the bundler.

Reply:
ah ok good to know